packages/@okta/vuepress-site/docs/reference/api/asa/groups/index.md

```diff
@@ -0,0 +1,619 @@
+| `group_name`   | string | The ASA Group name. |
+| `team_name`   | string | The name of your Team. |
+
+
+#### Request query parameters
+
+This endpoint has no query parameters.
+
+#### Request body
+
+This endpoint requires an object with the following fields.
+| Properties | Type        | Description          |
+|----------|-------------|----------------------|
+| `roles`   | array | A list of Roles for the ASA Group. Options are `access_user`, `access_admin` and `reporting_user`. |
+
+#### Response body
+This endpoint returns a `204 No Content` response on a successful call.
+
+
+#### Usage example
+
+##### Request
+
+```bash
+curl -v -X PUT \
+-H "Authorization: Bearer ${jwt}" \
+--data '{
+	"roles": [
+		"access_user",
+		"reporting_user",
+		"access_admin"
+	]
+}' \
+https://app.scaleft.com/v1/teams/${team_name}/groups/${group_name}
+```
+
+##### Response
+
+```json
+HTTP 204 No Content
+```
+### List ASA Users of an ASA Group
+
+<ApiOperation method="GET" url="https://app.scaleft.com/v1/teams/${team_name}/groups/${group_name}/users" />
+This endpoint lists all ASA Users in an ASA Group.
+
+This endpoint requires one of the following roles: `access_user`, `access_admin`, `reporting_user`.
+
+#### Request path parameters
+
+| Parameter | Type        | Description   |
+| --------- | ----------- | ------------- |
+| `group_name`   | string | The ASA Group name. |
+| `team_name`   | string | The name of your Team. |
+
+
+#### Request query parameters
+
+| Parameter | Type   | Description |
+| --------- | ------------- | -------- |
+| `contains`   |  string | (Optional) Includes ASA Users with name that contains the value. |
+| `count`   |  number | (Optional) The number of objects per page. |
+| `descending`   |  boolean | (Optional) The object order. |
+| `offset`   |  string | (Optional) The page offset. |
+| `prev`   |  boolean | (Optional) The direction of paging. |
+| `starts_with`   |  string | (Optional) Includes ASA Users with name that begins with the value. |
+| `status`   |  string | (Optional) Includes ASA Users with specified statuses. Valid statuses are `ACTIVE`, `DISABLED`, and `DELETED`. |
+| `user_type`   |  string | (Optional) Includes ASA Users of the specified type. Valid types are `human` and `service`. |
+
+
+#### Request body
+
+This endpoint has no request body.
+
+#### Response body
+This endpoint returns a list of objects with the following fields and a `200` code on a successful call.
+| Properties | Type        | Description          |
+|----------|-------------|----------------------|
+| `deleted_at`   | string | The time at which the ASA User was deleted. |
+| `details`   | object | An object with the following keys, the values of which are all strings: `first_name`, `last_name`, `full_name`, `email`. |
+| `id`   | string | The UUID of the ASA User. |
+| `name`   | string | The username of the ASA User. |
+| `oauth_client_application_id`   | string | The ID of the ASA User provided by OAuth, if it exists. |
```

Comment:
```suggestion
| `oauth_client_application_id`   | string | The ID of the ASA User provided by OAuth, if it exists |
```
Please do this for all `oauth_client_application_id` table entries in this section.